javascript/organizing_your_javascript_code/factory_functions_and_module_pattern.md

```diff
@@ -263,8 +263,10 @@ Take the calculator example into consideration. It's very easy to imagine a scen
 <div class="lesson-content__panel" markdown="1">
 
 1. WesBos has a beautiful and in-depth section on scopes and closures. Please check out these sections under "Module 3 - The Tricky Bits":
-  - [The article on scope](https://wesbos.com/javascript/03-the-tricky-bits/scope)
-  - [The article on closures](https://wesbos.com/javascript/03-the-tricky-bits/closures)
+
+- [The article on scope](https://wesbos.com/javascript/03-the-tricky-bits/scope)
+- [The article on closures](https://wesbos.com/javascript/03-the-tricky-bits/closures)
+
```

Comment:
```suggestion
   - [The article on scope](https://wesbos.com/javascript/03-the-tricky-bits/scope)
   - [The article on closures](https://wesbos.com/javascript/03-the-tricky-bits/closures)

```
I think the auto fixer may have accidentally misfixed this specific one, as it leads to a brand new list breaking up the ordered list.
Easy fix is to just indent the unordered list by 3 spaces.
Otherwise, everything else looks great!

Reply:
Good catch, it should be fixed now!